packages/lex-cli/src/codegen/server.ts

```diff
@@ -60,6 +78,30 @@ const indexTs = (project: Project, schemas: Schema[], nsidTree: NsidNS[]) =>
         .setNamespaceImport(toTitleCase(schema.id))
     }
 
+    // generate token enums
+    for (const nsidAuthority in nsidTokens) {
+      // export const {THE_AUTHORITY} = {
+      //  {Name}: "{authority.the.name}"
+      // }
+      file.addVariableStatement({
```

Comment:
I believe the type of `{THE_AUTHORITY}` is probably going to be of type `{ {Name}: string }`.  If an `as const` is added to the end then it would take its precise type, with the exact key-value pairs.  Could then go a step further and get a union of all the values, which might be a useful type, as `typeof {AUTHORITY}[keyof typeof {AUTHORITY}]`.

Reply:
Good call, I'll add the `as const`. Can you show a specific example of the latter suggestion though? I get the gist of it but I'm unsure what exactly you're wanting to construct.

Reply:
Yeah, sure thing!  Consider this type:
```ts
const Animal = {
  Dog: 'app.bsky.animalDog',
  Cat: 'app.bsky.animalCat',
} as const
```

If someone wanted to accept one of the app.bsky nsids for an animal as an argument then they'd want the type `'app.bsky.animalDog' | 'app.bsky.animalCat'`, which could be constructed from `Animal` as:
```ts
const AnimalNsid = typeof Animal[keyof typeof Animal]
//               = 'app.bsky.animalDog' | 'app.bsky.animalCat'
```

Reply:
Argh ts-morph is fighting me on adding `as const` too. How important is this?

Reply:
I don't think it's too important!  Still should be quite useful without it 🙏

Reply:
Oh yeah I see what you're saying. Yeah I think we'll just have to live without this for now.